### PR TITLE
Added sql file

### PR DESCRIPTION
**Pull request by kate-lvlup**

### CODE REVIEW

File: homework_29_280323/src/resources/init.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+DROP table persons;
+
+CREATE TABLE public.persons
+(
+    personId         bigint primary key generated by default as identity,
+    firstName        varchar(255),
+    lastName         varchar(255),
+    dateOfBirth      date,
+    salary           money,
```
Comment (YeaaaaH) [inline homework_29_280323/src/resources/init.sql:11]: use decimal(19,4) instead of money

File: homework_29_280323/src/resources/init.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
+CREATE database homework_29_280323;
+
+DROP table persons;
```
Comment (YeaaaaH) [inline homework_29_280323/src/resources/init.sql:3]: read about if exists statement

File: homework_29_280323/src/resources/init.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    dateOfBirth      date,
+    salary           money,
+    registrationDate timestamp with time zone
+);
+
+INSERT INTO persons(firstName, lastName, dateOfBirth, salary, registrationDate)
+values ('Lina', 'Kostenko', '1930-02-03', 20000, '2004-10-19 10:23:54 GMT');
+
+INSERT INTO persons(firstName, lastName, dateOfBirth, salary, registrationDate)
+values ('Lesya', 'Ukrainka', '1971-03-25', 19000.15, '2000-11-21 10:23:54 GMT'),
```
Comment (YeaaaaH) [inline homework_29_280323/src/resources/init.sql:19]: out them in 1 query

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
fix

File: homework_29_280323/src/resources/init.sql (hunk 1 above)
Comment (kate-lvlup) [reply homework_29_280323/src/resources/init.sql:11]: fixed

File: homework_29_280323/src/resources/init.sql (hunk 2 above)
Comment (kate-lvlup) [reply homework_29_280323/src/resources/init.sql:3]: I've read and fixed (removed DROP statement).

File: homework_29_280323/src/resources/init.sql (hunk 3 above)
Comment (kate-lvlup) [reply homework_29_280323/src/resources/init.sql:19]: Fixed.

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**